src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/codeSmell/UsageOfSilenceOperatorInspector.java

```diff
@@ -33,8 +34,8 @@
  */
 
 public class UsageOfSilenceOperatorInspector extends BasePhpInspection {
-    // configuration flags automatically saved by IDE
-    public boolean RESPECT_CONTEXT = true;
+    // Inspection options.
+    public boolean optionRespectContext = false;
```

Comment:
Yeah, I have checked that. But there are just few customizable inspections (_about 16 inspections_), then I belive that the user affected will be a small part. And from here to future we can keep the naming convension well.

About _spot magically saved options_, it is just the reason that I prefixed `option`. Then we know that `if(optionX)` is mean to be customizable "X" option. Use a uppercased variable seems that we working with a constant, and make confuses on first step.